### ISSUE TITLE
Rename Azure AD to Entra ID

### ISSUE THREAD

**Issue by superseb** [rancher/dashboard#9963]
- https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/new-name
- https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/how-to-rename-azure-ad

**Comment by richard-cox** [comment]
Re-opening, i think we still need this one

**Comment by richard-cox** [comment]
@codyrancher stale bot should probably avoid issues that are actively being processed (re-opened, moved stacks in project board, maybe even assigned?)

**Comment by codyrancher** [comment]
> [@codyrancher](https://github.com/codyrancher) stale bot should probably avoid issues that are actively being processed (re-opened, moved stacks in project board, maybe even assigned?)

Yeah, I think you're right about those first two items. I'll see if I can have the bot consider re-opened as an update and perhaps ignore project status >= review.

For assigned I think it's either a decent reminder to look at this or it's quick to nudge the bot to not close it.

**Comment by IsaSih** [comment]
Validated in 

[Rancher](https://github.com/rancher/rancher)	v2.15-8be5668519dba23417382428ab4acb72a98e88cb-head
[Dashboard](https://github.com/rancher/dashboard)	master df212f2
[Rancher CLI](https://github.com/rancher/cli)	v2.15.0-rc.1

Works as expected. Azure AD successfully renamed to Entra ID
